hashin.py

```diff
@@ -189,7 +298,18 @@ def main():
         default='sha256', nargs='?'
     )
     parser.add_argument(
-        "--verbose", help="Verbose output", action="store_true"
+        "-v, --verbose",
+        help="Verbose output",
+        action="store_true",
+        dest='verbose',
+    )
+    parser.add_argument(
+        '-p, --python-version',
+        help=('Python version to add wheels for. May be used multiple times. '
+              'Example: py2, cp35. Read as "Python 2" and "CPython 3.5".'),
```

Comment:
Oops. I forgot to update this bit of documentation after I added the version expansion. You could type py2.7, but it wouldn't get expanded.